Assets/_BForBoss/_Core/Scripts/UserInterface/BaseInputSettingsViewModel.cs

```diff
@@ -11,25 +12,25 @@ public abstract class BaseInputSettingsViewModel
         /// </summary>
         protected const float MAPPED_SENSITIVITY_MULTIPLIER = 100f;
         private readonly IBForBossAnalytics _analytics = null;
-        protected readonly IInputSettings _settings = null;
+        protected readonly IInputConfiguration _configuration = null;
         
         public abstract float GetHorizontal { get; }
         public abstract float GetVertical { get; }
         
-        public bool GetIsInverted => _settings.IsInverted;
+        public bool GetIsInverted => _configuration.IsInverted;
 
-        protected BaseInputSettingsViewModel(IInputSettings settings, IBForBossAnalytics analytics = null)
+        protected BaseInputSettingsViewModel(IInputConfiguration configuration = null, IBForBossAnalytics analytics = null)
```

Comment:
`IInputConfiguration configuration` this get's all the stored input configurations - sensitivty

And then shown on the UI